models/enterprise_sonic/mgmt_servers/sonic_mgmt_servers.yaml

```
@@ -71,6 +71,12 @@ DOCUMENTATION: |
                 type: str
                 choices:
                   - mgmt
+              cipher_suite:
+                description:
+                  - Cipher suites used for TLS connection with the clients
+                  - Specify as a comma separated list. Options are ecdhe-ecdsa-with-aes-256-gcm-SHA384, ecdhe-ecdsa-with-chacha20-poly1305-SHA256 and ecdhe-ecdsa-with-aes-128-gcm-SHA256.
+                type: str
+                  default: ecdhe-ecdsa-with-aes-256-gcm-SHA384,ecdhe-ecdsa-with-chacha20-poly1305-SHA256,ecdhe-ecdsa-with-aes-128-gcm-SHA256
```

The "default" option should be at the same indentation level as the "description" and "type" options.

Please fix the indentation of this line.
```suggestion
                default: ecdhe-ecdsa-with-aes-256-gcm-SHA384,ecdhe-ecdsa-with-chacha20-poly1305-SHA256,ecdhe-ecdsa-with-aes-128-gcm-SHA256
```